src/components/Reports/BadgeSummaryPreview.jsx

```diff
@@ -1,6 +1,7 @@
+// eslint-disable-next-line no-unused-vars
```

Comment:
I pulled your branch on my local and deleted this line 1 and ran `npx eslint <this_fine_name>`, found out it just complained about "React is defined but never used".

Please just delete your line 1 and delete the React import and apply such change to the repo.